src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -24,6 +26,7 @@ public OverseasEntitiesController(OverseasEntitiesService overseasEntitiesServic
 
     @PostMapping("/")
     public ResponseEntity<String> createNewSubmission(
+            @RequestAttribute("transaction") Transaction transaction,
             @RequestBody OverseasEntitySubmission overseasEntitySubmission) {
         ApiLogger.debug("Called createNewSubmission()");
 
```

Comment:
This change is fine, but we need to link the transaction to the new overseas entity submission (probably in the service call). We don't currently have a JIRA sub-task to do this. I'll add one if you agree @markpit ?

Reply:
yes we'll need to do it at some point, can be added as new task